components/CreateEventForm/CreateEventForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { StyleSheet, View, Button, Text, TextInput} from 'react-native';
+
+export default function CreateEventForm() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeButton}><Button title='X'/></View>
+      <Text style={styles.text}>CREATE A NEW EVENT FORM!</Text>
+      <TextInput style={styles.input} placeholder='Event Name...'/>
```

Comment:
Is this going to be the 'activity' drop down?

Reply:
I wasn't sure what to name it. We should discuss this further.